lib/ruby_lsp/internal.rb

```diff
@@ -30,15 +30,54 @@
 require "ruby_lsp/global_state"
 require "ruby_lsp/server"
 require "ruby_lsp/type_inferrer"
-require "ruby_lsp/requests"
-require "ruby_lsp/response_builders"
 require "ruby_lsp/node_context"
 require "ruby_lsp/document"
 require "ruby_lsp/ruby_document"
 require "ruby_lsp/erb_document"
 require "ruby_lsp/rbs_document"
 require "ruby_lsp/store"
 require "ruby_lsp/addon"
+
+# Response builders
+require "ruby_lsp/response_builders/response_builder"
+require "ruby_lsp/response_builders/collection_response_builder"
+require "ruby_lsp/response_builders/document_symbol"
+require "ruby_lsp/response_builders/hover"
+require "ruby_lsp/response_builders/semantic_highlighting"
+require "ruby_lsp/response_builders/signature_help"
+
+# Request support
+require "ruby_lsp/requests/support/rubocop_diagnostic"
+require "ruby_lsp/requests/support/selection_range"
+require "ruby_lsp/requests/support/annotation"
+require "ruby_lsp/requests/support/sorbet"
+require "ruby_lsp/requests/support/common"
+require "ruby_lsp/requests/support/formatter"
 require "ruby_lsp/requests/support/rubocop_runner"
 require "ruby_lsp/requests/support/rubocop_formatter"
 require "ruby_lsp/requests/support/syntax_tree_formatter"
+
+# Requests
+require "ruby_lsp/requests/request"
+require "ruby_lsp/requests/document_symbol"
```

Comment:
Let's order alphabetically?

Reply:
I ordered only the requests, but since there are dependencies between these files, we need to maintain the rest of the require order.

For example, `Request` needs to exist before we can inherit from that class, so it needs to be required first.